src/bin/scribble.ts

```diff
@@ -229,6 +233,7 @@ function instrumentFiles(
                 }
 
                 const annotations = gatherFunctionAnnotations(fun, annotMap);
+                annotations.concat(allowedFuncProp);
```

Comment:
Could you make sure here that contract-level `if_succeeds` functions are only added to the public and external functions?